lib/model/model.go

```diff
@@ -125,12 +130,14 @@ func NewModel(cfg *config.Wrapper, id protocol.DeviceID, deviceName, clientName,
 		folderRunners:     make(map[string]service),
 		folderStatRefs:    make(map[string]*stats.FolderStatisticsReference),
 		conn:              make(map[protocol.DeviceID]Connection),
-		deviceVer:         make(map[protocol.DeviceID]string),
+		config:            make(map[protocol.DeviceID]protocol.ClusterConfigMessage),
```

Comment:
Naming; we have lots of things that are various kinds of config. `deviceClusterConf` or `deviceCC` or something maybe?